proto/gz/msgs/inertial.proto

```diff
@@ -32,12 +32,41 @@ message Inertial
   /// \brief Optional header data
   Header header           = 1;
 
+  /// \brief Mass in kg
   double mass             = 2;
+
+  /// \brief CoM pose with respect to the link origin. In meters.
   Pose pose               = 3;
+
+  /// \brief Inertia matrix's XX element, in kg * m^2.
   double ixx              = 4;
+
+  /// \brief Inertia matrix's XY element, in kg * m^2.
   double ixy              = 5;
+
+  /// \brief Inertia matrix's XZ element, in kg * m^2.
   double ixz              = 6;
+
+  /// \brief Inertia matrix's YY element, in kg * m^2.
   double iyy              = 7;
+
+  /// \brief Inertia matrix's YZ element, in kg * m^2.
   double iyz              = 8;
+
+  /// \brief Inertia matrix's ZZ element, in kg * m^2.
   double izz              = 9;
+
+  /// \brief Fluid added mass matrix. The matrix is symmetric, so only the 21
+  /// elements of the top-half need to be set, organized as follows:
+  ///
+  /// 00, 01, 02, 03, 04, 05,
+  ///     11, 12, 13, 14, 15,
+  ///         22, 23, 24, 25,
+  ///             33, 34, 35,
+  ///                 44, 45,
+  ///                     55,
```

Comment:
this doesn't explicitly say how these values are packed into the `repeated double` vector. From looking at the test, I believe they are packed from left to right starting with the top row, then working down, like `[00, ... 05, 11, ... 15, 22, ... 25, 33, ... 35, 44, 45, 55]`

Reply:
What's throwing you off, the spacing? The elements are listed in that order exactly

Reply:
it looks like you are labeling the elements according to their place in the matrix, not listing the order in which they are placed in the vector

Reply:
it looks like the upper triangle of a matrix to me, not a vector

Reply:
Can you suggest the wording that would work for you? Maybe mentioning that the indices are row-first and instead of `organized as follows` say `ordered as follows`? Or something completely different?

Reply:
> it looks like the upper triangle of a matrix to me, not a vector

It's both! That's what I was going for at least...

Reply:
I guess it looks like a matrix to me and the indices are matrix indices, whereas vector indices would just be integers from [0..20]

~~~
 0,  1,  2,  3,  4,  5,
     6,  7,  8,  9, 10,
        11, 12, 13, 14,
            15, 16, 17,
                18, 19,
                    20,
~~~